api/v1beta1/hcpauth_types.go

```diff
@@ -42,7 +42,7 @@ type HCPAuthServicePrincipal struct {
 	//
 	//The secret data must have the following structure {
 	//   "clientID": "clientID",
-	//   "clientKey": "clientKey",
+	//   "clientSecret": "clientSecret",
```

Comment:
`clientKey` is mentioned on line 41 above as well.

Reply:
Fixed with 33639dab5df1fc80e44f4a312af681a2a08391c1